scripts/build_mirror_config.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Generate the sources.list.<arch> in the config path
+CONFIG_PATH=$1
+export ARCHITECTURE=$2
+export DISTRIBUTION=$3
+
+# The default mirror urls
+DEFAULT_MIRROR_URLS=http://debian-archive.trafficmanager.net/debian/,http://packages.trafficmanager.net/debian/debian/
+DEFAULT_MIRROR_SECURITY_URLS=http://debian-archive.trafficmanager.net/debian-security/,http://packages.trafficmanager.net/debian/debian-security/
+
+# The debian-archive.trafficmanager.net does not support armhf, use debian.org instead
+if [ "$ARCHITECTURE" == "armhf" ]; then
+    DEFAULT_MIRROR_URLS=http://deb.debian.org/debian/,http://packages.trafficmanager.net/debian/debian/
+    DEFAULT_MIRROR_SECURITY_URLS=http://security.debian.org/,http://packages.trafficmanager.net/debian/debian-security/
```

Comment:
I think we can use http://deb.debian.org/debian-security/ here for unification.

There are multiple variants:
security.debian.org
security.debian.org/debian-security
deb.debian.org/debian-security
and all work.

Related question: https://unix.stackexchange.com/questions/387006/sources-list-repository-url-for-debian-security-updates-with-or-without-debian

Documentation recommends to use deb.debian.org/debian-security or security.debian.org/debian-security.
Official debian docker images use deb.debian.org/debian-security since buster.

Reply:
> @xumia
> 
> Could you please retest a customization of Debian mirrors? I mean something like this:
> 
> `MIRROR_URLS="http://deb.debian.org/debian/,http://packages.trafficmanager.net/debian/debian/" MIRROR_SECURITY_URLS="http://deb.debian.org/debian-security,http://packages.trafficmanager.net/debian/debian-security/" make configure PLATFORM=vs`
> 
> `MIRROR_URLS="http://deb.debian.org/debian/,http://packages.trafficmanager.net/debian/debian/" MIRROR_SECURITY_URLS="http://deb.debian.org/debian-security,http://packages.trafficmanager.net/debian/debian-security/" make target/sonic-vs.img.gz`
> 
> I got correct sources.lists for sonic-slaves. But for docker-base-xxx and for rootfs (files/apt) I got debian-archive.trafficmanager.net instead of expected deb.debian.org.
> 
> I think you forgot to pass these variables from Makefile.work to slave.mk.

Fixed, thanks, missing to pass the variables.

Reply:
Fixed, "deb.debian.org/debian-security" is better than "security.debian.org".

Using the format as below:
```
http://deb.debian.org/debian/
http://deb.debian.org/debian-security/
```